console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/configuration/DeviceConfigPanel.java

```diff
@@ -454,7 +457,7 @@ public void handleEvent(BaseEvent be) {
             field.setMaxLength(1);
             field.setValidator((CharValidator) validator);
         }
-        if (validator != null && validator instanceof StringValidator) {
+        if (validator != null && validator instanceof StringValidator && field.getId().equals("WireGraph")) {
```

Comment:
This means that any `StringValidator` instance applied to a fild with id different than `WireGraph` will evaluate to false, and the validator will not be set. Please check this, and fix accordingly.